### DISCUSSION TITLE
☂️ Donating Funds to Biome

### DISCUSSION THREAD

**Discussion by Lfada** [biomejs/biome#1880] [General]
### Description

Hi Biome maintainers. I'm Lucas, a core contributor to the Drips project (https://www.drips.network/). I help recipients of donations through their learning and onboarding process.

[WAGMI](https://www.drips.network/app/projects/github/wevm/wagmi?exact) and [Abitype](https://www.drips.network/app/projects/github/wevm/abitype?exact) repo maintainers have been using https://github.com/biomejs/biome. It's in fact one of their most critical dependencies!

WAGMI and Abitype core contributors appreciate the work you all do and want to make sure that you are all well-funded to keep doing it. For that reason, they have decided to donate some funds to your project through [Drips](https://www.drips.network/).

Who can I talk to about it?

cc: @ematipico @Conaclos

**Comment by ematipico** [comment]
Not interested in crypto stuff, and I speak on behalf of the whole @biomejs/core-contributors

**Reply by Lfada** [reply]
Thanks for getting back to me. Will let the funder know, and they will hopefully direct the funds to another project.
